lib/ts/fetch.ts

```diff
@@ -668,8 +668,24 @@ export async function getLocalSessionState(tryRefresh: boolean): Promise<LocalSe
                     status: "NOT_EXISTS"
                 };
             }
-            logDebugMessage("getLocalSessionState: Retrying post refresh");
-            return await getLocalSessionState(tryRefresh);
+
+            const lastAccessTokenUpdate = await getFromCookies(LAST_ACCESS_TOKEN_UPDATE);
```

Comment:
this could come here if there is an API_ERROR output from refresh as well, but thats not handled here?

Reply:
It only gets here if the result is `RETRY` (see ln 664)